src/core/ActionMenu/ActionMenu.tsx

```diff
@@ -0,0 +1,240 @@
+import React, { forwardRef, FocusEvent, useState, useRef } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { AutoId } from '../utils/AutoId/AutoId';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../theme';
+import {
+  ActionMenuPopover,
+  InitialActiveDescendant,
+} from './ActionMenuPopover';
+import { Button, ButtonVariant } from '../Button/Button';
+import { HtmlDiv } from '../../reset';
+import { HTMLAttributesIncludingDataAttributes } from '../../utils/common/common';
+import { baseStyles } from './ActionMenu.baseStyles';
+import { ActionMenuItemProps } from './ActionMenuItem/ActionMenuItem';
+import { ActionMenuDividerProps } from './ActionMenuDivider/ActionMenuDivider';
+import { IconOptionsVertical } from 'suomifi-icons';
+
+const baseClassName = 'fi-action-menu';
+export const actionMenuClassNames = {
+  baseClassName,
+  fullWidth: `${baseClassName}--full-width`,
+  button: `${baseClassName}_button`,
+  iconOnly: `${baseClassName}_button--icon-only`,
+};
+
+export interface ActionMenuProps {
+  /** Text content for the button */
```

Comment:
Now that the list items are buttons as well, it might be good to differentiate between the menu button and the buttons inside the menu. So I'd clarify the "button" props so that the descriptions talk about menu button instead of just button.

Reply:
Now it's "the menu button". Should be easier to understand.